### ISSUE TITLE
Calculation of Fg (i.e. fraction escaping gut wall metabolism)

### ISSUE THREAD

**Issue by TWendl** [Open-Systems-Pharmacology/PK-Sim#517]
It could be realized in a similar way as the calculation of bioavailability as an extra button in "Show PK-Analysis".
In the background, a simulation should be created with absorption sink conditions (if not used as default) and portal vein blood blow set to zero. As a consequence, the drug is accumulating in the portal vein (plasma and blood cells). Fg, i.e. the fraction of drug escaping gut wall metabolism can be then calculated in the following way:
Fg = total amount of drug in portal vein (plasma + blood cells) / amount of administered drug

**Comment by AndreDlm** [comment]
Would be a very helpful feature in my opinion!

**Comment by msevestre** [comment]
@TWendl I don't think that what you are describing is `fraction escaping gut wall metabolism`
What you are describing is `Fraction absorved`

`fraction escaping gut wall metabolism` is the amount that enters portal vein divided by the amount that enters mucosa.  

fa = fg when all the dose enters mucosa

fg <=fa when there is solubility and or permeability limitation to uptake to mucosa

/cc @Yuri05

**Comment by StephanSchaller** [comment]
![image](https://user-images.githubusercontent.com/25477643/56638375-46192700-666e-11e9-8f8f-58f7ca55ca0e.png)

**Comment by StephanSchaller** [comment]
> fraction escaping gut wall metabolism is the amount that enters portal vein divided by the amount that enters mucosa.

I agree. The fraction "collected in PV" would need to be divided by fa.

**Comment by TWendl** [comment]
I agree, you are right! In my particular case,  it always was fa=1 .

**Comment by AndreDlm** [comment]
Any news when this will be implemented? It should be rather simple to implement and would be a nice feature :)

@Yuri05 @msevestre @Erik-Sjogren

**Comment by msevestre** [comment]
Specify it and I can implement it...

**Comment by AndreDlm** [comment]
> Specify it and I can implement it...

Well, as suggested by @TWendl:
first pass Fg = (total amount of drug in portal vein (plasma + blood cells) / amount of administered drug) / fa


@sfrechen @StephanSchaller Any comments?

**Comment by StephanSchaller** [comment]
Would it help if @AndreDlm creates the observer in MoBi and provides it as a PKML? ... maybe this can facilitate things

**Comment by msevestre** [comment]
Do we need to run another simulation as proposed? Or  do we have the Info required already?

**Comment by StephanSchaller** [comment]
Yes, I guess, if you mean the one @TWendl described above ("...portal vein blood flow set to zero...").

**Comment by AndreDlm** [comment]
Can we get again the prio label? :) Maybe even `prio:high`?

**Comment by msevestre** [comment]
I cannot implement this feature at the moment based solely on the description above. It's probably obvious for everyone but me but since I will probably be the one to implement it...I need to be sure this is properly specified.

Specifically: @Yuri05 Do we have all the parts of the equations above already? Or do we need to create new observers to compute them? @AndreDlm and @Yuri05 @TWendl. WE can also setup a meeting for next week

**Comment by NinaHanke** [comment]
Hej hej, 

Does that mean that you are working on generating Fg as a simulation output?  

A reviewer wants me to provide Fa (available), Fg, Fh. 
If we have Fg, Fh could then be calculated from Fh = Bioavailability / (Fa * Fg). 

![F = Fa x Fg x Fh](https://github.com/user-attachments/assets/c86b9d5a-abcc-409e-bc01-e457797c3776)

Happy New Year! 
Nina

**Comment by NinaHanke** [comment]
Hey there, 
Could somebody tell me please, if the following formula is correct? 

Fraction unbound in gut enterocytes = fu,gut =  fu,plasma / partition coefficient between intracellular space mucosa and plasma 

Best, 
Nina
